### ISSUE TITLE
[Health 11.1.0] improvement: add fall detection events

### ISSUE THREAD

**Issue by tobiasbader** [cph-cachet/flutter-plugins#1067]
[Health 11.1.0]

Description:
On iOS, HealthKit provides the possibility to get fall detection events from the Apple Watch. 
On Android, the Pixel Watch also has fall detection. But I am not sure if it is possible to read these alerts via Google Health.
It would be really great if this plugin would include the option for fall detection events as an additional HealthDataType similar to the already existing HealthDataType HIGH_HEART_RATE_EVENT.
Thanks a lot.

**Comment by manuelbirkner** [comment]
I completely agree – having fall detection events as an additional HealthDataType would be amazing! Thanks a lot for bringing this up! 🙌

**Comment by capc0307** [comment]
I think having fall detection events would be awesome and incredible helpful. Consider including this feature, please.
